llvm/test/ThinLTO/X86/memprof-tailcall-nonunique.ll

```diff
@@ -0,0 +1,161 @@
+;; Test to make sure that missing tail call frames in memprof profiles are
+;; identified but not cloned when there are multiple non-unique possible
+;; tail call chains between the profiled frames.
+
```

Comment:
I didn't end up including that here because I did a lot of manual clean up of the original test (for example there were calls to other functions like sleep and delete etc). So I'm not convinced there is a lot of value in including that here.